### PR TITLE
Roman add phpunit retry lib

### PR DESCRIPTION
**Pull request by fejjo**
Jira: KBC-XXX
Connection PR: XXXXX

Before asking for review make sure that:

- [x] New client method(s) has tests
- [x] Apiary file is updated
- [x] You declared if there is a BC break or not (will affect next release of a client)

---
Urobim composer update na mastru

Ak si checkoutnem tuto branch testy musia stale robit retry 3x
vytvorim si novy file z phpunit-retry.xml.dist a pomenujem ho phpunit-retry.xml ked spustim teraz testy neurobi to retry ani jeden krat.
mozem si lubovolne upravit pocet retry pre lokal.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -64,6 +64,8 @@ You can run these tests only against non-production environments.
 
 Tests requires valid Keboola Management API tokens and an endpoint URL of the API test environment.
 
+*Note: For automated tests, the tests are run again three times by default if they fail. For local development this would be quite annoying, 
+so you can disable this by creating new file `phpunit-retry.xml` from `phpunit-retry.xml.dist`*
```
Comment (jirkasemmler) [inline README.md:68]:
```suggestion
so you can disable this by creating new file `phpunit-retry.xml` from `phpunit-retry.xml.dist`*

```

Comment (jirkasemmler) [reply README.md:68]: aby se to odradkovalo s tou dalsi note. Ono to jinak spoji do jednoho odstavce

**Review by jirkasemmler — APPROVED** [review]
prosim jen submitni to moje suggestion kvuli citelnosti, jinak je to cajk, fakci to skvele. Udelas to prosim i do sapi? :heart:

**Comment by fejjo** [comment]
@jirkasemmler jasne pridam tom tam cakal som na schvalenie nech to nemusim upravovat na dvoch miestach rovnaku vec dik. Commitol som tvoju zmenu

**Pull request merged**